keep-trying

```diff
@@ -138,19 +139,21 @@ else
         if [[ -z "$email_subject" ]] ; then
             email_subject="$(basename $0) unsuccessful"
         fi
-        # Set sender email address
+        # Set sender email address(es)
         if [[ ! -z "$log_file" ]] ; then
             if [[ $zip_log -eq 1 ]] ; then
                 zip -rq "${log_file}.zip" $log_file
                 log_file="${log_file}.zip"
             fi
-            echo -e $email_body | mutt -s "$email_subject" $email_address -a "$log_file"
+            #echo -e $email_body | mutt -s "$email_subject" $email_address -a "$log_file"
```

Comment:
Can you remove this commented out code, and the others below?

Reply:
Done.